### DISCUSSION TITLE
General feedback

### DISCUSSION THREAD

**Discussion by littleK0i** [littleK0i/SnowDDL#1] [General]
Welcome!

Please post your general feedback about SnowDDL here. Please create a new topic for specific suggestions.

Thank you.

**Comment by marksabincarnivaluk** [comment]
Hi,
Thank you so much for creating this tool. I have been struggling with Terraform and all its idiosyncrasies before discovering SnowDDL. It addresses so many of the issues I was having with Terraform, and it is much easier to create and maintain configuration than with imperative style script and tools.

**Reply by oz-chewie** [reply]
Should have read "declarative style".

**Comment by oz-chewie** [comment]
Added a PR for task conversion here:
https://github.com/littleK0i/SnowDDL/pull/66

**Reply by littleK0i** [reply]
@oz-chewie , please note, I've just added some additional parameters to TaskConverter:

https://github.com/littleK0i/SnowDDL/blob/master/snowddl/converter/task.py#L53-L64

Things might change a little bit if you update and run it again.
